Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "busybody"
+version = "0.1.0"
+authors = ["Levi Pearson <levipearson@gmail.com>"]
+
+[lib]
+name = "busybody"
+path = "src/lib.rs"
+
+[[bin]]
```

Comment:
Observation: You can create a `bin` folder in the `src` directory and each individual file in there will be compiled as a separate binary.

Reply:
Cool, I wasn't aware of that one. I am not married to the approach of calling them out explicitly in the Cargo.toml, but I kind of like having the explicit target.